app/models/user.rb

```diff
@@ -4,9 +4,9 @@ class User < ApplicationRecord
   include RandomToken
 
   has_many :authorization_requests, dependent: :destroy
-  has_many :jwt_api_entreprise, through: :authorization_requests
+  has_many :token, through: :authorization_requests
```

Comment:
Je suis peu fan du singularize ici. imo faut pluralize

```suggestion
  has_many :tokens, through: :authorization_requests
```